[2021-03-17 18:08:00,592] INFO [GroupMetadataManager brokerId=3] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-03-17 18:18:00,602] INFO [GroupMetadataManager brokerId=3] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-03-17 18:48:01,556] INFO [GroupMetadataManager brokerId=3] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-03-17 18:48:01,630] INFO [GroupMetadataManager brokerId=3] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-03-17 18:48:03,402] WARN Client session timed out, have not heard from server in 1799510ms for sessionid 0x1783a1c8f440016 (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:04,397] INFO Client session timed out, have not heard from server in 1799510ms for sessionid 0x1783a1c8f440016, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:01,781] WARN [ReplicaFetcher replicaId=3, leaderId=1, fetcherId=0] Error in fetch to broker 1, request (type=FetchRequest, replicaId=3, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={my-kafka-topic2-1=(offset=3, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 1 was disconnected before the response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:95)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:96)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:205)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:41)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:149)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:113)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:64)
[2021-03-17 18:48:01,761] INFO [GroupCoordinator 3]: Member consumer-1-4751e914-ec82-4539-8699-aabbfbfe3276 in group console-consumer-70126 has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:08,111] INFO [GroupCoordinator 3]: Preparing to rebalance group console-consumer-70126 with old generation 16 (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:08,211] INFO [GroupCoordinator 3]: Group console-consumer-70126 with generation 17 is now empty (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:01,650] WARN [ReplicaFetcher replicaId=3, leaderId=2, fetcherId=0] Error in fetch to broker 2, request (type=FetchRequest, replicaId=3, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={my-kafka-topic2-0=(offset=25, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 2 was disconnected before the response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:95)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:96)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:205)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:41)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:149)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:113)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:64)
[2021-03-17 18:48:08,984] INFO [GroupCoordinator 3]: Preparing to rebalance group console-consumer-70126 with old generation 17 (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:08,419] INFO [GroupMetadataManager brokerId=3] Removed 0 expired offsets in 4358 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-03-17 18:48:07,231] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:09,470] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:05,861] INFO zookeeper state changed (Disconnected) (org.I0Itec.zkclient.ZkClient)
[2021-03-17 18:48:09,254] INFO [GroupCoordinator 3]: Stabilized group console-consumer-70126 generation 18 (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:09,861] INFO zookeeper state changed (Expired) (org.I0Itec.zkclient.ZkClient)
[2021-03-17 18:48:09,948] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=org.I0Itec.zkclient.ZkClient@49ec71f8 (org.apache.zookeeper.ZooKeeper)
[2021-03-17 18:48:09,838] WARN Unable to reconnect to ZooKeeper service, session 0x1783a1c8f440016 has expired (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:09,989] INFO Unable to reconnect to ZooKeeper service, session 0x1783a1c8f440016 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:09,988] INFO EventThread shut down for session: 0x1783a1c8f440016 (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:09,985] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:10,012] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:10,035] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x1783a1c8f44001a, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2021-03-17 18:48:10,044] INFO zookeeper state changed (SyncConnected) (org.I0Itec.zkclient.ZkClient)
[2021-03-17 18:48:10,048] INFO [GroupCoordinator 3]: Preparing to rebalance group console-consumer-70126 with old generation 18 (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:10,060] INFO re-registering broker info in ZK for broker 3 (kafka.server.KafkaHealthcheck$SessionExpireListener)
[2021-03-17 18:48:10,077] INFO Creating /brokers/ids/3 (is it secure? false) (kafka.utils.ZKCheckedEphemeral)
[2021-03-17 18:48:10,107] INFO Result of znode creation is: OK (kafka.utils.ZKCheckedEphemeral)
[2021-03-17 18:48:10,110] INFO Registered broker 3 at path /brokers/ids/3 with addresses: EndPoint(127.0.0.1,9095,ListenerName(PLAINTEXT),PLAINTEXT) (kafka.utils.ZkUtils)
[2021-03-17 18:48:10,112] INFO done re-registering broker (kafka.server.KafkaHealthcheck$SessionExpireListener)
[2021-03-17 18:48:10,117] INFO Subscribing to /brokers/topics path to watch for new topics (kafka.server.KafkaHealthcheck$SessionExpireListener)
[2021-03-17 18:48:19,960] INFO [GroupCoordinator 3]: Member consumer-1-6072e77c-9e8b-4d01-bb71-1a13a05280d9 in group console-consumer-70126 has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:19,972] INFO [GroupCoordinator 3]: Stabilized group console-consumer-70126 generation 19 (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:48:19,976] INFO [GroupCoordinator 3]: Assignment received from leader for group console-consumer-70126 for generation 19 (kafka.coordinator.group.GroupCoordinator)
[2021-03-17 18:58:00,592] INFO [GroupMetadataManager brokerId=3] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
